packages/frontend/src/features/core/services/ReportService/ReportService.ts

```diff
@@ -48,7 +47,7 @@ export class ReportService extends RelayApiService {
         type: ReportType
         objectId: string
         reason: string
-        category: ReportCategory
+        category: number
```

Comment:
why number instead of ReportCategory? then this would be magic numbers? (same as the below ones)

Reply:
Because report categories come from relay API, the value of category may not be inconsistent with ReportCategory type. This is a lazy solution that we don't bother category value in frontend. This value should be validated in the relay server.